components/camera/server_test.go

```diff
@@ -35,24 +35,26 @@ var (
 	errCameraUnimplemented      = errors.New("not found")
 )
 
-func newServer() (pb.CameraServiceServer, *inject.Camera, *inject.Camera, *inject.Camera, error) {
+func newServer() (pb.CameraServiceServer, *inject.Camera, *inject.Camera, *inject.Camera, *inject.Camera, error) {
```

Comment:
Can we use one of the other cameras injected in this tests to test the same functionality? Seems like a lot to add a fourth camera to test framerates are properly returned from the Properties API.

Reply:
Got it - going to use injectDepthCamera to test not setting frame rate instead of making another camera